### ISSUE TITLE
Build is failing on CircleCI

### ISSUE THREAD

**Issue by fedorov** [QIICR/dcmqi#255]
@jcfr something changed recently, and CircleCI started to fail in this step:

```
# Install tools required to run DCMQI "doc" tests
cd /home/ubuntu/dcmqi/docker/.. && \
	/tmp/dockcross bash -c "export PATH=/work/build/node-v6.9.5-linux-x64/bin:$PATH && npm install jsonlint -g && npm install ajv-cli -g"
npm ERR! Linux 3.13.0-117-generic
npm ERR! argv "/work/build/node-v6.9.5-linux-x64/bin/node" "/work/build/node-v6.9.5-linux-x64/bin/npm" "install" "jsonlint" "-g"
npm ERR! node v6.9.5
npm ERR! npm  v3.10.10
npm ERR! path /home/ubuntu/.npm
npm ERR! code EACCES
npm ERR! errno -13
npm ERR! syscall mkdir

npm ERR! Error: EACCES: permission denied, mkdir '/home/ubuntu/.npm'
npm ERR!     at Error (native)
npm ERR!  { Error: EACCES: permission denied, mkdir '/home/ubuntu/.npm'
npm ERR!     at Error (native)
npm ERR!   errno: -13,
npm ERR!   code: 'EACCES',
npm ERR!   syscall: 'mkdir',
npm ERR!   path: '/home/ubuntu/.npm' }
npm ERR!
npm ERR! Please try running this command again as root/Administrator.

npm ERR! Please include the following file with any support request:
npm ERR!     /work/npm-debug.log
make: *** [prereq.install_npm_packages] Error 243
```
Running `npm install jsonlint -g` without `dockcross` works.

I tried [with](https://circleci.com/gh/QIICR/dcmqi/648) and [without](https://circleci.com/gh/QIICR/dcmqi/649) cache.

Do you know what is going on?

**Comment by fedorov** [comment]
If I run this line with sudo: `/tmp/dockcross bash -c "export PATH=/work/build/node-v6.9.5-linux-x64/bin:$PATH && npm install jsonlint -g && npm install ajv-cli -g"`, I get past that error, but then I get another error:

```
Error response from daemon: Driver btrfs failed to remove root filesystem e30dd3455fd008180cd4c5ff2336d154a17002673ed60e186d6f68d57973cd2d: Failed to destroy btrfs snapshot /var/lib/docker/btrfs/subvolumes for e30dd3455fd008180cd4c5ff2336d154a17002673ed60e186d6f68d57973cd2d: operation not permitted
Error: failed to remove containers: [dockcross_6348]
make: *** [prereq.install_npm_packages] Error 1
```

**Comment by fedorov** [comment]
I set this to high priority, because with CircleCI failing, we will not get updated Linux binaries, and will not get updated Docker images, so it is very important to have this addressed sometime soon!

**Comment by fedorov** [comment]
The same "tree" worked fine on CircleCI for this build: https://circleci.com/gh/QIICR/dcmqi/644

**Comment by fedorov** [comment]
Running `npm` with `sudo` seems to get rid of the error. Will test this by making a PR:

`$(TMP)/dockcross bash -c "export PATH=/work/build/node-v6.9.5-linux-x64/bin:$$PATH && sudo npm install jsonlint -g && sudo npm install ajv-cli -g"`

**Comment by fedorov** [comment]
Resolved via #256
